tsql/TSqlLexer.g4

```diff
@@ -807,7 +807,7 @@ LINE_COMMENT:       '--' ~[\r\n]* -> channel(HIDDEN);
 DOUBLE_QUOTE_ID:    '"' ~'"'+ '"';
 SINGLE_QUOTE:       '\'';
 SQUARE_BRACKET_ID:  '[' ~']'+ ']';
-LOCAL_ID:           '@' ([A-Z_$@#0-9] | FullWidthLetter)+;
+LOCAL_ID:           '@' ([A-Za-z_$@#0-9] | FullWidthLetter)+;
```

Comment:
`a-z`

Remove it. We use UPPER case insensitive input stream, see [caseInsensitiveType](https://github.com/antlr/antlr4test-maven-plugin#caseinsensitivetype)

Reply:
As as user of the grammar I am not really interested in the antlr4test-maven-plugin plugin. How do I use the grammar to parse lower case parameter names without modifying the Lexer rule?

Reply:
You can use case-insensitive input stream in any runtime, see [Runtimes](https://github.com/KvanTTT/DAGE/tree/master/AntlrGrammarEditor/Runtimes).

As a T-SQL grammar creator, I'm not interested in dirty and slow grammar due to the fragment rules.